workflow/custom-dashboard/dashboard-builder-overview.mdx

```diff
@@ -0,0 +1,127 @@
+
+#### Group By
+
+The Group By field is the outer most aggregation made in your query. The way in which you aggregate a metric is contingent on the initial metric you have selected for your widget. Group your initial metric with resource attributes, metric attributes, or another metric based on the selected metric. The below example is grouping the `container.name` metric by all unique instances of the `container.name` metric:
+
+![](https://lh7-us.googleusercontent.com/lJ_3zm4C37Zgv8oWBRza4Z9-BbdJdlXD-lxbFq2qfGD579JQ9tNR_1MbC0zLLLW9ufQ-MQQa1_nwD4tg5PG-9QTpCDdsX5rE4UDIN-y9sILxyPlCHF8C8FcYv1od9VqK2ZACHS1NBAQFDk6yuOLgRzM)
+
+The following table displays all available methods for aggregating your chosen metrics:
+
+| __Aggregation Type__ | __Description__                                                |
+| :------------------- | :------------------------------------------------------------ |
+| __Average By__       | Average of all values in selected time range for given metric |
+| __Sum By__           | Sum of all values in selected time range for given metric |
+| __Minimum By__       | Minimum of all values in selected time range for given metric |
+| __Maximum By__       | Maximum of all values in selected time range for given metric |
+| __Any By__           | Gathers all possible values in selected time range for given metric __and/or__ attribute |
+| __Unique By__        | Count of unique values in selected time range for given metric __and/or__ attribute |
+| __Count By__         | Count of all values in selected time range for given metric __and/or__ attribute |
+| __Group By__         | Groups all values in selected timerange for given metric __and/or__ attribute |
+
+#### Inner Value Aggregation
+
+The Inner Value Aggregation is a nested aggregation technique that allows you to aggregate your initial query. The below example is grouping the `k8s.node.memory.usage` metric by summing the `k8s.cluster.name` attributes together and aggregating __Any__ `k8s.node.memory.usage`:
+
+![](https://lh7-us.googleusercontent.com/fo5Img7dsK14nN-w6atI390lqE-Lrg9oHS6BlqlgLwJKjNjpke85oDeEVYfhbl3CGj3hPiZ6mfnK5XXGPZGlxbfzY5wjd31myEhAR4gAJnQgB5Eii1Ex7CvkQgufEmUNFVcKZiLkUBFt35WBIGRtBHc)
+
+#### Filter
+
+The filter feature is used to remove unwanted metrics within your query. The below example is filtering out any `host.id` containing `ip-172-31-13-218.us-east-2.compute.internal`:
+
+![](https://lh7-us.googleusercontent.com/_50CgdVxjgsqeqS6U3S8Rud9llopVXSi97d2WXCo4shRYJp79HjUs8gGTq_ZLihDDlP4wpi3IqCvaT6z4NpnhXAQWzhNWbe2uUS89Gw4Nd8ww_RX95Ce5p4LsR_SBO3-0LSp3sAht3heL01QDKgCAQQ)
+
+#### Advanced
+
+The Advanced section allows you to specify the interval at which your data is ingested and displayed at.
```

Comment:
Similar to other sections, let's have a list of supported functions for this area.